### ISSUE TITLE
White screen on screen sleep

### ISSUE THREAD

**Issue by konn-neko** [Genymobile/scrcpy#2849]
- [yes ] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [ yes] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: [Windows 10]
 - scrcpy version: [1.20]
 - installation method: [Git]
 - device model: Tinno u304ac ???
 - Android version: [9 go edditon (shitty phone)]

**Describe the bug**
when pressing the power button on phone with the -Sw option, it gives a white screen every time the power is pressed or emulated

```
this might just be a phone problem
```

**Comment by rom1v** [comment]
Thank you for your report. Probably a device issue, scrcpy just requests to change the "power mode" to the device. How the device behaves in this mode is out of scrcpy control.

**Comment by maleeqB** [comment]
Only smooth experience with experience with scrcpy is on Android 7 Nougat.
Since then
Tecno BC2c (Android 10)
Tecno KF7j (Android 11) 

Once scrcpy is connected, the screen doesn't turn it off completely (Not immediately after being connected, maybe after screen on/off two or three times). Disconnecting scrcpy isn't enough until I do a full reboot


Dunno if this is what op is experiencing, also funny others aren't experiencing this.

**Comment by rom1v** [comment]
> Disconnecting scrcpy isn't enough until I do a full reboot

Even pressing the physical power button several times is not sufficient?

**Comment by maleeqB** [comment]
Yes, pressing the physical power button several times is not sufficient.
